db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/BinaryLogicalOperatorNode.java

```diff
@@ -100,18 +100,9 @@ public ValueNode bindExpression(FromList fromList,
 	 *
 	 * @return		Boolean which reflects validity of the tree.
 	 */
-	boolean verifyEliminateNots()
-	{
-		if (SanityManager.ASSERT)
-		{
-			return (leftOperand.verifyEliminateNots() &&
-					rightOperand.verifyEliminateNots());
-		}
-		else
-		{
-			return true;
-		}
-	}
+	boolean verifyEliminateNots() {
+        return !SanityManager.ASSERT || (leftOperand.verifyEliminateNots() && rightOperand.verifyEliminateNots());
```

Comment:
I would keep this pattern as it was, it's easier to see what's part of the assertion and what's not. If there are other cases please revert them too.